### ISSUE TITLE
Change default shortcuts for comment/uncomment to M1+7 (Ctrl+7)

### ISSUE THREAD

**Issue by bugybunny** [eclipse/texlipse#45]
Default shortcut for commenting lines is now Ctrl+5 and uncomment Esc, Ctrl+5. Default Eclipse shortcut everywhere else is Ctrl+7. Ctrl+7 in a TeX file does not seem to be doing anything else, so changing to Ctrl+7 does not seem to create a conflict with something. 

Not sure if that would be easy, but having the same shortcut for a toggle comment function would be cool, also like everywhere else in Eclipse.

Btw: I will finally try to commit something here in around January, have too much else to do right now. I really really like Texlipse, use it a lot and would love to help or implement something real quick and not only report issues.

**Comment by turesheim** [comment]
Thanks Marco. I've added the "help wanted" label as this issue should be a really straightforward fix.

**Comment by bugybunny** [comment]
Just saw that #36 is related or maybe even the same. However, I have never used  CTRL+SHIFT+C for toggle comment. Maybe these two issues can be merged.

**Comment by turesheim** [comment]
Good catch. I think the keybinding(s) for commenting out lines and the operation's actual behaviour are two different issues. I'd prefer that we did not merge these.

**Comment by gerking** [comment]
The good thing about Ctrl+5 is that the 5 key directly refers to the % sign that is meant to be written. For the same reason, JDT supports Ctrl+/ to comment/uncomment lines in Java (beside Ctrl+7). If Ctrl+7 is really a default, I'm not voting against it. But please don't drop support for Ctrl+5, also for legacy reasons.
